draft-pda-protocol.md

```diff
@@ -670,13 +675,19 @@ parameterized by a PAParam. At a high level, it proceeds as follows. First, the
 collector initializes local per-protocol state using the corresponding
 PAParam. This state object has the following member functions:
 
-- CreateRequest() -> PACollectReq: Creates a PACollectReq object, defined below,
-  that is sent to the leader to complete one iteration of the protocol.
+- CreateRequest(): Creates a PACollectReq object, defined below, that is sent to
+  the leader to complete one iteration of the protocol. Along with any
+  protocol-specific parameters, the message specifies a time interval
+  `[batch_start, batch_end)` that determines the batch of reports to be
+  aggregated. It must be that `batch_end - batch_start >= PAParam.batch_window`
+  and `batch_start` and `batch_end` are multiples of `PAParam.batch_window`.
```

Comment:
I like making the collector specify the batch window, as I think it absolves us of specifying some tricky edge cases w.r.t. insufficient # of reports in the window (we can just say that it's up to the collector to try again with wider batch windows until they find one that works).

Reply:
Or maybe the collector can try again with a smaller window if it wants a different slice?